src/test/java/org/mockitousage/junitrule/StubbingWarningsJUnitRuleTest.java

```diff
@@ -0,0 +1,257 @@
+package org.mockitousage.junitrule;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.Statement;
+import org.mockito.Mock;
+import org.mockito.internal.junit.JUnitRule;
+import org.mockito.internal.util.SimpleMockitoLogger;
+import org.mockitousage.IMethods;
+import org.mockitoutil.TestBase;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class StubbingWarningsJUnitRuleTest extends TestBase {
+
+    private SimpleMockitoLogger logger = new SimpleMockitoLogger();
+    private JUnitRule jUnitRule = new JUnitRule(logger, false);
+    private FrameworkMethod dummy = mock(FrameworkMethod.class);
+
+    @After public void after() {
+        //so that the validate framework usage exceptions do not collide with the tests here
+        resetState();
+    }
+
+    @Test
+    public void no_unused_stubs_reported_on_failure() throws Throwable {
+        try {
```

Comment:
Better use the Rule [ExpectedException ](http://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html)to verify the expected exception.